source/index.js

```diff
@@ -163,6 +164,14 @@ module.exports = async (input = 'patch', options) => {
 		]);
 	}
 
+	if (runStatusChecks) {
+		tasks.add([{
+			title: 'Ensuring GitHub status checks have passed',
+			enabled: () => isOnGitHub === true,
```

Comment:
I'd move this check to line 52 (the declaration of `runStatusChecks`).

Reply:
That also works 👍🏻

Reply:
Changed in 0b54e4e2cb67d8908d0bdec9174c01729ccde147.